platform/projects_and_organizations/members.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: members
+title: Members
+sidebar_label: Members
+---
+
+Once you have created a project and set up the languages, you can add members to your project. Members are the people who will be translating your project.

```

Comment:
And what about the user's who are managing the project? I don't think we have to explain who is a member. If you wish to do so, you can say "it's an user who has permissions to read or modify project data such as translations, keys, etc.".